<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuxBmsTab</class>
 <widget class="QWidget" name="AuxBmsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="allowCharge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Allow Charge</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="auxBmsAlive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Aux BMS Alive</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="strobeBmsLight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Strobe BMS Light</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="auxVoltage">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Max: 14.5 V; Lowest 10.8 V&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="prechargeStateLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Precharge State</string>
   </property>
  </widget>
  <widget class="QLabel" name="auxVoltageLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Max: 14.5 V; Lowest 10.8 V&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Aux Voltage</string>
   </property>
  </widget>
  <widget class="QComboBox" name="prechargeState">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COMMON_ENGAGED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CHARGE_ENGAGED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DISCHARGE_ENGAGED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALL_ENGAGED</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="highVoltageEnableState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>High Voltage Enable State</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="allowDischarge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Allow Discharge</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="orionCanReceivedRecently">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Orion CAN Received Recently</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chargeContactorError">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge Contactor Error</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dischargeContactorError">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Discharge Contactor Error</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="commonContactorError">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Common Contactor Error</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dischargeShouldTrip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Discharge Should Trip</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chargeShouldTrip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge Should Trip</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chargeOpenButShouldBeClosed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge Open But Should Be Closed</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dischargeOpenButShouldBeClosed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>290</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Discharge Open But Should Be Closed</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="auxTripGroupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Aux Trip</string>
   </property>
   <widget class="QCheckBox" name="chargeTripDueToHighCellVoltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>290</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge Trip Due to High Cell Voltage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chargeTripDueToHighTemperatureAndCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge Trip Due to High Temperature and Current</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dischargeTripDueToHighTemperatureAndCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Discharge Trip Due to High Temperature and Current</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chargeTripDueToPackCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge Trip Due to Pack Current</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dischargeTripDueToLowCellVoltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Discharge Trip Due to Low Cell Voltage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dischargeTripDueToPackCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Discharge Trip Due To Pack Current</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="protectionTrip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Protection Trip</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
